src/LuaTransformer.ts

```diff
@@ -132,12 +133,13 @@ export class LuaTransformer {
             this.popScope();
 
             if (this.isModule) {
+                const exportsTable = !this.visitedExportEquals ? tstl.createTableExpression() : undefined;
+
                 // local exports = {}
+                // or
+                // local exports
```

Comment:
Considering that `export =` can be used only once and only on top level, maybe it would be better to declare a local on the export assignment itself?